src/line_cursor.ts

```diff
@@ -175,6 +175,27 @@ export class LineCursor extends Marker {
     return newNode;
   }
 
+  /**
+   * Returns true iff the node to which we would navigate if in() were
+   * called, which will be a validInLineNode, is also a validLineNode
+   * - in effect, if the LineCursor is at the end of the 'current
+   * line' of the program.
+   *
+   * This is used to determine whether to display an "Edit Block
+   * contents (→︎)" entry in the block context menu, to help users
+   * discover that the right arrow key can be used to navigate to
+   * block contents.
```

Comment:
Nit, but this might be an overly specific documentation item as it's concerning itself with how this function may be used (which also may not stay true forever if context menu logic changes or we find new uses for this functionality).

Reply:
It was intending to explain the _purpose_ of this otherwise somewhat trivial-seeming action.  Perhaps you could recommend a better wording?